frcnn_detector.py

```diff
@@ -262,17 +279,27 @@ def detect(self, img):
         dummy_labels = np.zeros((self.__nr_rois, self.labels_count))
 
         # prepare the arguments
-        arguments = {
-            self.__model.arguments[self.__args_indices["features"]]: [img_model_arg],
-            self.__model.arguments[self.__args_indices["rois"]]: [test_rois]
-        }
+        if (self.__isPythonModel):#python model

```

Comment:
Instead of doing this if, you can just set the values of self.__args_indices["features"] and self.__args_indices["rois"] when you load the python model and then this code path can remain unchanged.